src/layouts/EditHomepage/constants.ts

```diff
@@ -0,0 +1,41 @@
+export const RESOURCES_SECTION = {
+  title: "Resources",
+  subtitle: "Add a preview and link to your Resource Room",
+  id: "resources",
+} as const
+
+export const INFOBAR_SECTION = {
+  title: "Infobar",
+  subtitle: "Add informational text",
+  id: "infobar",
+  description: "Infobar description",
+  button: "Button Text",
+  url: "",
+} as const
+
+export const INFOPIC_SECTION = {
+  title: "Infopic",
+  subtitle: "Add an image and text",
+  id: "infopic",
+  description: "Infopic description",
+  button: "Button Text",
+  url: "", // No default value so that no broken link is created
+  image: "", // Always blank since the image modal handles this
+  alt: "Image alt text",
+} as const
+
+export const KEY_HIGHLIGHT_SECTION = {
+  title: "Key Highlight Title",
+  description: "Key Highlight description",
+  url: "", // No default value so that no broken link is created
+} as const
+
+export const DROPDOWN_ELEMENT_SECTION = {
+  title: "Hero Dropdown Element Title",
+  url: "", // No default value so that no broken link is created
+} as const
+
+export const DROPDOWN_SECTION = {
+  title: "Hero Dropdown Title",
+  options: [],
+} as const
```

Comment:
do we need the as const?

Reply:
yeap - the `as const` is to tell `ts` that this is a `readonly const` value. there's 2 differences with this cast

1. type inference becomes alot stricter - w/ the cast, `DROPDOWN_SECTION.title` is always inferred as `Hero dropdown title` and not `string` 
2. no reassignment of properties - with the cast, we cannot do `DROPDOWN_SECTION.title = "mutated"`; without the cast, we **can**. the `const DROPDOWN_SECTION` applies only to the _identifier_, which means that we can't do `const DROPDOWN_SECTION = ... // something with same shape` but can still do reassignment of properties

Reply:
cool, makes sense

Reply:
@seaerchin good explanation, using `as const` enforces literal types and read-only properties